packages/tap-runner/test/integration/tap-test-runner.it.spec.ts

```diff
@@ -208,4 +208,33 @@ describe('tap-runner integration', () => {
       assertions.expectCompleted(run);
     });
   });
+
+  describe('Running on a ava project', () => {
+    beforeEach(async () => {
+      sandbox = new TempTestDirectorySandbox('ava');
+      await sandbox.init();
+      const avaLocation = path.resolve(sandbox.tmpDir, '..', '..', '..', 'node_modules', 'ava', 'entrypoints', 'cli.mjs');
+      options.tap = tapRunnerOptions({
+        nodeArgs: [avaLocation, '--tap'],
+      });
+      sut = testInjector.injector.injectFunction(createTapTestRunnerFactory('__stryker2__'));
+      await sut.init();
+    });
+
+    it('should be able to run', async function () {
+      // Test fails if debugger is used, because ava doesn't support debugging and tap output simultaneously.
+      // See: https://github.com/avajs/ava/blob/568fe40c987dd6c593dfbcf4144d1d1627955d46/lib/cli.js#L307
+      const { default: inspector } = await import('node:inspector');
+
```

Comment:
This is not needed. What we should do instead is disable `"autoAttachChildProcesses"` in launch.json. See https://rcrdo.com/2018/08/07/favorite-new-visual-studio-code-features-of-2018/#_node-debugging